src/components/frame/TokenActivation/TokensTab/TokensTab.tsx

```diff
@@ -161,23 +145,28 @@ const TokensTab = ({
             </TokenTooltip>
             <div className={styles.tokenNumbers}>
               <span data-test="activeTokens">
-                <Numeral value={formattedActiveTokens} suffix={token.symbol} />
+                <Numeral
+                  value={activeBalance ?? 0}
+                  decimals={tokenDecimals}
+                  suffix={nativeToken.symbol}
+                />
               </span>
-              {formattedActiveTokens === SMALL_TOKEN_AMOUNT_FORMAT && (
-                <SmallTokenAmountMessage />
-              )}
```

Comment:
I felt like these are not relevant anymore. The `Numeral` component will display the small values using engineering notation. And I guess the UI refresh will bring some clarification in regards to number formatting rules